pkg/app/mysql-deploymentconfig.go

```diff
@@ -55,7 +55,7 @@ type MysqlDepConfig struct {
 func NewMysqlDepConfig(name string) App {
 	return &MysqlDepConfig{
 		name:       name,
-		dbTemplate: getOpenShiftDBTemplate(mysqlDepConfigName),
+		dbTemplate: getOpenShiftDBTemplate(mysqlDepConfigName, ephemeralStorage),
```

Comment:
We should be to choose persistent storage type also. We can add a field in `MysqlDepConfig` and refer it